pkg/v1/tkg/web/src/app/shared/components/error-import-file/error-import-file.component.html

```diff
@@ -0,0 +1,15 @@
+<div *ngIf="errorImportFile" class="alert alert-danger" role="alert">
```

Comment:
Would it be possible to have a single import-file-notification component that takes an additional input for notification type (success and error) or a notification is an object with error message and notification type?

We could adjust the css classes applied using and expression with ngClass, or the notification component ts file could have a little logic to code the class and icon. How does that sound?